packages/spacecat-shared-rum-api-client/src/functions/opportunities/high-inorganic-high-bounce-rate.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import trafficAcquisition from '../traffic-acquisition.js';
+import { getCTRByUrl } from '../../common/aggregateFns.js';
+
+function convertToOpportunity(traffic) {
+  const { url, total, bounceRate } = traffic;
+
+  return {
+    type: 'high-inorganic-high-bounce-rate',
+    page: url,
+    screenshot: '',
+    trackedPageKPIName: 'Bounce Rate',
+    trackedPageKPIValue: bounceRate,
+    pageViews: total,
+    samples: total, // todo: get the actual number of samples
+    metrics: [], // none needed
+  };
+}
+
+function hasHighInorganicTraffic(traffic) {
+  const { url, paid, total } = traffic;
+  const isHomapage = new URL(url).pathname === '/';
+  const threshold = isHomapage ? 0.5 : 0.8;
```

Comment:
Would there be a need for those to be site-specific and passed as configuration or query parameters somehow? or can we keep these static values for all

Reply:
better to make these as configurable as env variables

Reply:
I extracted those values to constant.

For now, I'd say let's start with constants. Later-on we can make it configurable per site if needed. This should be data-driven, I really don't know what values best for each customer at the moment.